### ISSUE TITLE
Allow maintainers to view/edit reports as plain text

### ISSUE THREAD

**Issue by mvz** [418sec/huntr#2146]
Not all researchers understand markdown and some of the reports I get are hard to read due to very bad formatting in markdown. It makes sense that I can't edit them but switching to plain wrapped text would help a lot.

**Comment by JamieSlome** [comment]
Ooo, nice suggestion! 👏 

Just tagging @psmoros for his thoughts here ❤️

**Comment by psmoros** [comment]
Makes sense! Markdown confuses me too 😅 I will weigh the effort / impact of this feature this week and update @mvz :)
